src/components/MultipleAvatars.tsx

```diff
@@ -1,77 +1,85 @@
-import PropTypes from 'prop-types';
 import React, {memo, useMemo} from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import {ValueOf} from 'type-fest';
+import {AvatarSource} from '@libs/UserUtils';
 import styles from '@styles/styles';
 import * as StyleUtils from '@styles/StyleUtils';
 import themeColors from '@styles/themes/default';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import Avatar from './Avatar';
-import avatarPropTypes from './avatarPropTypes';
 import Text from './Text';
 import Tooltip from './Tooltip';
 import UserDetailsTooltip from './UserDetailsTooltip';
 
-const propTypes = {
+type AvatarType = {
```

Comment:
please check if the type `Avatar` doesn't exist in the code base, I think someone added that type.
Also following ts migration guide, shouldn't use 'type' suffix in type decalration

Reply:
Yep, already resolved!